### ISSUE TITLE
My shapes are no longer displayed in 0.2.0

### ISSUE THREAD

**Issue by NiklasEi** [Nilirad/bevy_prototype_lyon#68]
First of all: thanks for this awesome crate!

I have been drawing shapes with version 0.1.5 and decided to update to 0.2.0 today. After fixing all compilation issues, the Entities are spawned, but nothing is visible on the screen. Example code:

```rust
    let rectangle = shapes::Rectangle {
        width: 18.0,
        height: 18.0,
        ..shapes::Rectangle::default()
    };
    let mut builder = GeometryBuilder::new();
    builder.add(&rectangle);
    let mut enemy = Enemy {
      // some stuff
    };
    commands
        .spawn(builder.build(
            enemy.get_color_handle(materials),
            TessellationMode::Fill(FillOptions::default()),
            Transform::from_translation(Vec3::new(map.spawn.x, map.spawn.y, 0.)),
        ))
        .with(enemy);
```
        
Am I missing something? In case you would like to run it yourself: https://github.com/NiklasEi/oicana/tree/bevy_prototype_lyon_v0.2.0

**Comment by Nilirad** [comment]
Hi!
I tried to run your game on Windows in the branch you specified, but as soon as I launch it says
```
error: process didn't exit successfully: `target\debug\oicana.exe` (exit code: 0xc000041d)
```
I did a bit of research but I couldn't find much.

About your shapes not being displayed, even if I can't run it, I'm 99% sure that's because you didn't include the `ShapePlugin`. Also keep in mind that you put your game logic into the `"oicana_stage"`. However, in my library there is a [SHAPE](https://docs.rs/bevy_prototype_lyon/0.2.0/bevy_prototype_lyon/plugin/stage/constant.SHAPE.html) stage, that runs after Bevy's `UPDATE` stage that uses a plugin system to add a mesh to the `ShapeBundle`. I think `"oicana_stage"` should run before the `SHAPE` stage to avoid unexpected problems or off by one frame spawning.

TLDR:
1. Add `ShapePlugin`
2. Add `"oicana_stage"` before `bevy_prototype_lyon::plugin::stage::SHAPE`

**Comment by NiklasEi** [comment]
Yes, thank you, adding the plugin fixed most of my issues.One more needed change was not centring arcs at (0, 0)
